packages/core/admin/admin/src/pages/Settings/pages/ApiTokens/EditView/EditViewPage.tsx

```diff
@@ -173,6 +175,23 @@ export const EditView = () => {
     }
   }, [data]);
 
+  React.useEffect(() => {
+    // Only set up timer when token is shown
+    if (showToken) {
+      hideTimerRef.current = setTimeout(() => {
+        setShowToken(false);
+      }, 30000); // 30 seconds
```

Comment:
So it auto hides after 30 seconds and that's the only way of hiding the token?

For me being able to manually close/hide it seems like a better experience? Or
maybe having both so we still have the added security of auto hiding ?

Reply:
we can have both, just following the acceptance criteria
@maccomaccomaccomacco can make the decision on that

Reply:
Yeah, could the "View" button toggle to "Hide"?

Reply:
yeah it currently auto hides, the question was if we want the button to also hide on/off, basically a show and hide button, it's a very simple addition to add for me if we choose to go that way

Reply:
I don't think it's important, it could come as a later improvement. The user can always navigate away if they want it hidden quickly

Reply:
let me check with Louis if he can come out with something very quick; but if you need to wait for this thing only in order to approve the PR, let's skip it and let's handle it later

Reply:
talked to Louis, we'll have it also hide, and have a hide eye icon

Reply:
wait, what do you mean with "we'll have it also hide"?